control-plane-operator/controllers/hostedcontrolplane/v2/assets/ingress-operator/deployment.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ingress-operator
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      name: ingress-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: ingress-operator
+        name: ingress-operator
+    spec:
+      containers:
+      - command:
+        - ingress-operator
+        - start
+        - --namespace
+        - openshift-ingress-operator
+        - --image
+        - $(IMAGE)
+        - --canary-image
+        - $(CANARY_IMAGE)
+        - --release-version
+        - $(RELEASE_VERSION)
+        - --metrics-listen-addr
+        - 0.0.0.0:60000
+        env:
+        - name: RELEASE_VERSION
+          value: ""
+        - name: IMAGE
+          value: ""
+        - name: CANARY_IMAGE
+          value: ""
+        - name: KUBECONFIG
+          value: /etc/kubernetes/kubeconfig
+        - name: HTTP_PROXY
+          value: http://127.0.0.1:8090
+        - name: HTTPS_PROXY
+          value: http://127.0.0.1:8090
+        - name: NO_PROXY
+          value: kube-apiserver
+        image: cluster-ingress-operator
+        imagePullPolicy: IfNotPresent
+        name: ingress-operator
+        resources:
+          requests:
+            cpu: 10m
+            memory: 80Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - mountPath: /etc/kubernetes
+          name: ingress-operator-kubeconfig
+        - mountPath: /var/run/secrets/openshift/serviceaccount
+          name: serviceaccount-token
+      - args:
+        - --service-account-namespace=openshift-ingress-operator
+        - --service-account-name=ingress-operator
+        - --token-file=/var/run/secrets/openshift/serviceaccount/token
+        - --kubeconfig=/etc/kubernetes/kubeconfig
+        command:
+        - /usr/bin/control-plane-operator
+        - token-minter
+        image: token-minter
+        imagePullPolicy: IfNotPresent
+        name: token-minter
+        ports:
+        - containerPort: 60000
+          name: metrics
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        volumeMounts:
+        - mountPath: /var/run/secrets/openshift/serviceaccount
+          name: serviceaccount-token
+        - mountPath: /etc/kubernetes
+          name: admin-kubeconfig
+      volumes:
+      - name: ingress-operator-kubeconfig
```

Comment:
ingress kubeconfig can be self service after https://github.com/openshift/hypershift/pull/5607